src/core/configProvider/configContext.ts

```diff
@@ -22,6 +23,7 @@ export type Config = {
       props: React.AnchorHTMLAttributes<HTMLAnchorElement>
     ) => JSX.Element;
     Head?: (props: { children: React.ReactNode }) => JSX.Element;
+    CollectionCard?: (props: CollectionItemType) => JSX.Element;
```

Comment:
I decided not to use a new key for mapping this component, because if I would have done so:

1. We would have multiple contexts to configure components
2. Configuration object would have get nested levels (which are not needed yet and so the configuration is clear and simple).
3. With a CollectionCard the devs can already make their own collection / item specific card decisions. CollectionCard component should be a quite simple one from the implementation perspective.